### DISCUSSION TITLE
Display the Bitcoin price and Fear & Greed Index on the RaspiBlitz status screen

### DISCUSSION THREAD

**Discussion by neversleep404** [mempool/mempool#4553] [Ideas]
Although the status screen is full, maybe a UX designer can help to meaningfully display the current Bitcoin price and the Fear & Greed Index. Perhaps it would be sensible to switch every minute between a graphical Bitcoin index including Fear & Greed and the practical status screen?

**Comment by softsimon** [comment]
Fiat price is something we have in an API. But Fear and greed is an outside arbitrary metric that is not related to the Mempool project.
